src/runtime_src/core/common/dev_factory.h

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (C) 2023 Advanced Micro Devices, Inc. All rights reserved.
+#ifndef _XCL_DEV_FACTORY_H_
+#define _XCL_DEV_FACTORY_H_
+
+#include "device.h"
+#include <vector>
+
+namespace xrt_core {
+
+// One function on FPGA or AIE device
+class dev
+{
+public:
+  bool m_is_mgmt = false;
+  bool m_is_ready = true; 
+  dev(bool isuser) {  
+      m_is_mgmt = !isuser;  
+  }
```

Comment:
What's with the public data members, and does the concept of mgmt / user continue to be of use to XRT coreutil?  Can we maybe eliminate it?

Reply:
we can move them to private and can access through public member functions, currently we are using  m_is_mgmt flag while creation of dev_factory class, like core/pcie/pcidev_linux.cpp has some functionality based on this.